src/app/feature/map/search/search-bar/search-bar.component.html

```diff
@@ -0,0 +1,61 @@
+<form class="select" [formGroup]="formGroup">
+  <div class="icon-wrapper">
+    <svg class="icon">
+      <use [attr.xlink:href]="svgIconUrl"></use>
+    </svg>
+  </div>
+  <mat-form-field class="form-field" appearance="legacy">
+    <mat-label class="label">{{ labelText | translate }}</mat-label>
+    <input
+      class="input"
+      type="text"
+      matInput
+      formControlName="value"
+      [matAutocomplete]="autoGroup"
+    />
+    <mat-autocomplete #autoGroup="matAutocomplete">
+      <mat-option [value]="''">{{ "UI.CANCEL" | translate }}</mat-option>
+      <mat-optgroup
+        *ngFor="let group of stateGroupOptions | async"
+        [label]="group.groupName"
+      >
+        <mat-option *ngFor="let room of group.rooms" [value]="room.viewValue">
+          {{ room.viewValue }}
+        </mat-option>
+      </mat-optgroup>
+    </mat-autocomplete>
+  </mat-form-field>
+  <button type="submit" class="submit-btn" (click)="findLocation()">
+    <svg
```

Comment:
Please investigate, can we move svg tags into shared component, for now template is very huge